pkg/config/runtime_test.go

```diff
@@ -85,4 +89,5 @@ func TestNewRuntimeConfig(t *testing.T) {
 	_, err = NewRuntimeConfig(k0sVars)
 	assert.Error(t, err)
 	assert.ErrorIs(t, err, ErrK0sAlreadyRunning)
+	require.NoError(t, spec.Cleanup())
```

Comment:
Is that some cleanup stuff? Then I'd use `t.Cleanup(...)` at an appropriate place, just as you'd be using defer.

Reply:
done.